src/router/index.js

```diff
@@ -16,6 +17,10 @@ export default new Router({
     {
       path: '/test',
       component: TestPage
+    },
+    {
+      path: '/coupon',
+      component: CouponPage
     }
```

Comment:
I think the commit message should be `Add coupon page routes`. Fix it next time ya